ui-admin/src/portal/dashboard/widgets/StudyWidget.tsx

```diff
@@ -0,0 +1,114 @@
+import { InfoCard, InfoCardBody, InfoCardHeader } from 'components/InfoCard'
+
+export const StudyWidget = ({ portal }: { portal: Portal }) => {
+  const [showNewStudyModal, setShowNewStudyModal] = useState(false)
+  const { user } = useUser()
+
+  const primaryStudy = portal.portalEnvironments.find(env =>
+    env.environmentName === 'live')?.portalEnvironmentConfig.primaryStudy
+
+  const sortedStudies = portal.portalStudies.sort((a, b) => {
+    if (a.study.shortcode === primaryStudy) { return -1 }
+    if (b.study.shortcode === primaryStudy) { return 1 }
+    return a.study.name.localeCompare(b.study.name)
+  })
+
+  return (
+    <InfoCard>
+      <InfoCardHeader>
+        <div className="d-flex align-items-center justify-content-between w-100">
+          <span className="fw-bold">Studies</span>
+          <Button onClick={() => setShowNewStudyModal(true)}
+            disabled={!user?.superuser}
+            tooltip={user?.superuser ? 'Create a new study' : 'You do not have permission to create a new study'}
+            variant="light" className="border">
+            <FontAwesomeIcon icon={faPlus} className="fa-lg"/> Create
+          </Button>
+        </div>
+        {showNewStudyModal && <CreateNewStudyModal portal={portal} onDismiss={() => setShowNewStudyModal(false)}/>}
+      </InfoCardHeader>
+      <InfoCardBody>
+        <ul className="list-group list-group-flush w-100">
+          {sortedStudies.map((portalStudy, i) => {
+            const study = portalStudy.study
+            return <StudyControls
+              key={i}
+              portal={portal}
+              study={study}
+              primaryStudy={primaryStudy}
+            />
+          })}
+        </ul>
+      </InfoCardBody>
+    </InfoCard>
+  )
+}
+
+const StudyControls = ({ portal, study, primaryStudy }: {
+  portal: Portal,
+  study: Study,
+  primaryStudy?: string,
+}) => {
+  const { reload } = useNavContext()
+  const [showDeleteStudyModal, setShowDeleteStudyModal] = useState(false)
+
+  return (
+    <li key={`${portal.shortcode}-${study.shortcode}`}
+      className="list-group-item my-1 border border-secondary-subtle rounded py-1">
+      <div className="d-flex justify-content-between align-items-center">
+        <div className="py-2">
+          <Link to={studyParticipantsPath(portal.shortcode, study.shortcode, 'live')}>
+            <img
+              src={getMediaUrl(portal.shortcode, 'favicon.ico', 'latest')}
+              className="me-2" style={{ maxHeight: '1.5em' }} alt={study.name}/>
+            {study.name}
+          </Link>
+          {study.shortcode === primaryStudy &&
+            <span className="badge bg-secondary ms-2">Primary Study</span>}
+        </div>
+        <div className="dropdown">
+          <Button
+            data-bs-toggle='dropdown'
+            className="dropdown-toggle border m-1"
+            type="button"
+            id={`editStudyMenu-${study.shortcode}`}
```

Comment:
why does this button need an id?

Reply:
hmmm, im not sure, this part was copied over from the previous study list component. doesn't seem to be necessary. i'll remove it